yt_dlp/extractor/servus.py

```diff
@@ -1,11 +1,9 @@
 from .common import InfoExtractor
+from .. import traverse_obj
 from ..utils import (
-    determine_ext,
     float_or_none,
     int_or_none,
-    unified_timestamp,
-    urlencode_postdata,
-    url_or_none,
+    unified_timestamp, ExtractorError,
 )
```

Comment:
Make sure to sort the imports, format them correctly (isort?) and also import `traverse_obj` from `..utils`.

Reply:
Probably something misconfigured in IntelliJ, sorry - will fix now

Reply:
Fixed in f21384ac68c9845c7be912576f59dd55bf594c87

By the way, if this is important we should probably add this guideline somewhere, did not see isort mentioned in the guidelines, and running it in my project changed 520 extractors 😄 but that is another topic

Reply:
Coding guidelines and formatting is only loosely enforced by the reviewers, and not strictly defined as well. There are more than one way to do it "correctly", though I would like to have some automation for this as well. Maybe this could be tracked in another issue though 😄

Reply:
> By the way, if this is important we should probably add this guideline somewhere, did not see isort mentioned in the guidelines, and running it in my project changed 520 extractors 😄 but that is another topic

You are only supposed to run it on the extractor being worked on. And even if you didn't, it's not really a blocker since I can just pull the PR and run isort locally (hence why I haven't added it to contributing guidelines)

As for automation - for ease of mergeability, I want to get the convention into ytdl first before we enforce it here. See https://github.com/ytdl-org/youtube-dl/pull/31350